FatmaQunnies/classes/task.py

```diff
@@ -0,0 +1,32 @@
+from datetime import date
+from task_status import Status
+class Task:
+ 
+    counter = 0  
+    def __init__(self, title, description, due_date, status =Status.IN_PROGRESS):
+        Task.counter += 1 
+        self.task_id = Task.counter 
+        self.title = title
+        self.description = description
+        self.due_date = due_date
+        self.status = status
+
+    def update_details(self, title=None, description=None, due_date=None,status=None):
```

Comment:
Same thing here: use one object to take all variables and iterate over it using loop